third_party/move/move-compiler/tests/move_check/parser/constants_simple.exp

```diff
@@ -1,30 +1,6 @@
-error[E04013]: invalid statement or expression in constant
-  ┌─ tests/move_check/parser/constants_simple.move:5:21
-  │
-5 │     const C1: u64 = u();
-  │                     ^^^ Module calls are not supported in constants
-
-error[E04013]: invalid statement or expression in constant
-  ┌─ tests/move_check/parser/constants_simple.move:7:22
-  │
-7 │     const C3: bool = loop ();
-  │                      ^^^^^^^ 'loop' expressions are not supported in constants
-
-error[E04013]: invalid statement or expression in constant
-  ┌─ tests/move_check/parser/constants_simple.move:8:20
-  │
-8 │     const C4: u8 = if (cond) 0 else 1;
-  │                    ^^^^^^^^^^^^^^^^^^ 'if' expressions are not supported in constants
-
-error[E03009]: unbound variable
-  ┌─ tests/move_check/parser/constants_simple.move:8:24
-  │
-8 │     const C4: u8 = if (cond) 0 else 1;
-  │                        ^^^^ Invalid variable usage. Unbound variable 'cond'
-
-error[E04013]: invalid statement or expression in constant
-  ┌─ tests/move_check/parser/constants_simple.move:9:38
-  │
-9 │     const C5: vector<vector<bool>> = abort 0;
-  │                                      ^^^^^^^ 'abort' expressions are not supported in constants
+error[E01007]: invalid byte string
+   ┌─ tests/move_check/parser/constants_simple.move:16:29
+   │
+16 │     const C12: vector<u8> = "<empty>";
+   │                             ^ String literal must begin with b" (for a byte string) or h" (for a hex string)
```

Comment:
I think we should break this test up, so that this new blocking (parse) error does not shadow the compile errors being showcased before.

Reply:
Done.